tox.ini

```diff
@@ -60,6 +60,7 @@ deps =
     pydocstyle
 commands =
     flake8 src/coronawhy_vt/ setup.py
+    flake8 tests/ setup.py
```

Comment:
put `tests/` on the first line so there's only one set of feedback in this run like:

```
commands =
    flake8 src/coronawhy_vt/ tests/ setup.py
```